src/EnergyPlus/SQLiteProcedures.cc

```diff
@@ -1816,11 +1817,13 @@ void SQLite::createSQLiteDaylightMap(int const mapNum,
 {
     if (m_writeOutputToSQLite) {
         ++m_hourlyReportIndex;
-        sqliteBindInteger(m_daylightMapHourlyTitleInsertStmt, 1, m_hourlyReportIndex);
-        sqliteBindForeignKey(m_daylightMapHourlyTitleInsertStmt, 2, mapNum);
-        sqliteBindInteger(m_daylightMapHourlyTitleInsertStmt, 3, month);
-        sqliteBindInteger(m_daylightMapHourlyTitleInsertStmt, 4, dayOfMonth);
-        sqliteBindInteger(m_daylightMapHourlyTitleInsertStmt, 5, hourOfDay);
+        int b = 0;
```

Comment:
Sure, it's reasonable to use a counter here instead of manually persisting the order.